node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -605,4 +606,10 @@ public void createSymlink(Path symlink, Path target) {
     protected Optional<CredentialsMaintainer> credentialsMaintainer() {
         return credentialsMaintainer;
     }
+
+    private static Duration warmUpDurationForZone(ZoneApi zone) {
+        return zone.getSystemName() == SystemName.cd || zone.getEnvironment().isTest()
```

Comment:
Any reason this shouldn't apply to `PublicCd`?
```suggestion
        return zone.getSystemName().isCd() || zone.getEnvironment().isTest()
```